lib/local_system.rb

```diff
@@ -43,6 +43,39 @@ def validate_existence_of_package(package)
       end
     end
 
+    def validate_existence_of_packages(packages)
```

Comment:
Assignment Branch Condition size for validate_existence_of_packages is too high. [15.39/15]